src/components/menu/menu.ts

```diff
@@ -162,22 +163,24 @@ export class Menu extends Component<Props, SpreadsheetChildEnv> {
     return this.menuComponentHeight(menusAbove) + this.position.y;
   }
 
-  private onClick(ev: MouseEvent) {
+  private onClick(ev: MenuMouseEvent) {
```

Comment:
If I do not miss something, the two functions (onClick and onContextMenu) are the same functions.